test/sql/include/join_query.sql

```diff
@@ -2160,6 +2160,7 @@ rollback to settings;
 savepoint settings;
 set local max_parallel_workers_per_gather = 0;
 set local work_mem = '128kB';
+set enable_mergejoin to false;
```

Comment:
You're disabling mergejoin here, but do we have similar merge joins in other tests?

Reply:
this test requires hashjoin and doesnt apply to mergejoin